README.adoc

```diff
@@ -160,6 +160,15 @@ Assuming you've installed `libmagic` properly and you have
 https://golang.org/doc/install[setup a Go environment], Use the
 following command to install Pistol to `$GOPATH/bin/pistol`:
 
+==== Go 1.17 or later
+
+[source,sh]
+----
+go install github.com/doronbehar/pistol/cmd/pistol@latest
+----
+
+==== Go 1.16 or earlier
```

Comment:
How new is `go install`? Perhaps this section should be removed..

Reply:
Go 1.17 (which introduced "go install" and deprecated "go get") was released August 16, 2021. I just checked and saw that Go 1.17 and 1.18 are both available in the official repositories of the current Ubuntu LTS. What do you think?

Reply:
> UPDATE: I just double-checked the deprecation documented, and, actually, "go install" was introduced in Go 1.16 (Feb 16 2021). Go 1.17 is when "go get" was deprecated.

Thanks for the thorough check, that's appreciated. If that's so, let's change this line to:

```suggestion
==== Go 1.15 or earlier
```

Reply:
Done